public/script.js

```diff
@@ -0,0 +1,249 @@
+import { getCategories, getToken, getPlaylists, search } from './api.js';
+
+// html элемент, в котором находится основной контент на странице
+const content = document.querySelector('.content');
+// поисковая строка
+const input = document.querySelector('.header__search');
+
+/** 
+ * Функция для создания заготовки раздела страницы
+ * @returns html элемент section, состоящего из h2 - заголовка и div - контейнера для хранения элементов
+ */
+function createSection() {
+  let section = document.createElement('section');
+  section.className = 'section';
+  const template = `
+  <h2 class="section__header"></h2>
+  <div class="items"></div>
+  `;
+  section.insertAdjacentHTML('beforeend', template);
+  return section;
+}
+
+/**
+ *  Функция для создания элемента раздела страницы на основе переданного объекта 
+ * @returns ссылку, состоящую из изображения и подписи
+*/
+function objectToHtmlElement(obj) {
+  const a = document.createElement('a');
+  a.tabIndex = 0;
+  a.classList = 'item link footer__item-link';
```

Comment:
стоит еще добавить
```suggestion
  a.target= '_blank';
```

так при нажатии на карточку, будет открываться новая вкладка.
На мой взгляд так удобнее.